script/publishing-api-sync-checks/case_study_sync_check.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../", File.dirname(__FILE__))
+
+require 'pathname'
+require 'logger'
+logger = Logger.new(STDERR)
+logger.info "Booting rails..."
+require '../../config/environment'
+logger.info "Booted"
+
+case_studies = CaseStudy.latest_published_edition
+check = DataHygiene::PublishingApiSyncCheck.new(case_studies)
+check.add_expectation("format") do |content_store_payload, model|
+  content_store_payload["format"] == 'case_study'
+end
+check.add_expectation("base_path") do |content_store_payload, model|
+  content_store_payload["base_path"] == model.search_link
+end
+check.add_expectation("title") do |content_store_payload, model|
+  content_store_payload["title"] == model.title
+end
+check.perform
```

Comment:
I know it's not available here because it's not in content store, but we should work out a way to check the rendering app too else we won't know that something has been fully migrated away from Whitehall.